### PR TITLE
Remove underscore prefix from grpc_options parameter

### PR DESCRIPTION
**Pull request by DavidCurtiss**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

- [x] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.

~~- [ ] I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?

Renames the optional `_grpc_options` Session parameter to `grpc_options`

### List issues fixed by this Pull Request below, if any.

None

### What testing has been done?

Unit tests, etc.

### CODE REVIEW

**Review by ni-jfitzger — APPROVED** [review]
The change looks good to me.
Test failures are due to an issue with the setup of our VM that we should resolve.

**Review by dmondrik — APPROVED** [review]
LGTM

**Review by bkeryan — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ All notable changes to this project will be documented in this file.
 * ### ALL
     * #### Added
         * Support for Python 3.10
+        * MeasurementLink support
```
Comment (marcoskirsch) [inline CHANGELOG.md:36]: Let's be more precise and only put this line in those drivers that actually got the support added. I believe the list is:

✅ nidcpower
✅ nidigital
✅ nidmm
✅ nifgen
⛔️ nimodinst
✅ niscope
⛔️ nise
✅ niswitch
⛔️ nitclk

**Review by marcoskirsch — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (marcoskirsch) [reply CHANGELOG.md]: did it

**Review by marcoskirsch — APPROVED** [review]

**Pull request merged**